smack-experimental/src/main/java/org/jivesoftware/smackx/stanza_content_encryption/element/PayloadElement.java

```diff
@@ -0,0 +1,54 @@
+/**
+ *
+ * Copyright 2020 Paul Schaub
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jivesoftware.smackx.stanza_content_encryption.element;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.jivesoftware.smack.packet.ExtensionElement;
+import org.jivesoftware.smack.packet.NamedElement;
+import org.jivesoftware.smack.packet.XmlEnvironment;
+import org.jivesoftware.smack.util.XmlStringBuilder;
+
+public class PayloadElement implements NamedElement {
+
+    public static final String ELEMENT = "payload";
+
+    private final List<ExtensionElement> payloadElements;
+
+    public PayloadElement(List<ExtensionElement> payloadElements) {
+        this.payloadElements = Collections.unmodifiableList(payloadElements);
+    }
+
+    public List<ExtensionElement> getItems() {
+        return payloadElements;
+    }
+
+    @Override
+    public String getElementName() {
+        return ELEMENT;
+    }
+
+    @Override
+    public XmlStringBuilder toXML(XmlEnvironment xmlEnvironment) {
+        XmlStringBuilder xml = new XmlStringBuilder(this).rightAngleBracket();
+        for (ExtensionElement element : payloadElements) {
```

Comment:
Could simply be `xml.append(payloadElements)`.

Reply:
ah, some syntactic sugar I didn't know yet :)